daemon/info_unix.go

```diff
@@ -233,8 +233,9 @@ func parseInitVersion(v string) (version string, commit string, err error) {
 func parseRuncVersion(v string) (version string, commit string, err error) {
 	lines := strings.Split(strings.TrimSpace(v), "\n")
 	for _, line := range lines {
-		if strings.HasPrefix(line, "runc version") {
-			version = strings.TrimSpace(strings.TrimPrefix(line, "runc version"))
+		if strings.Contains(line, "version") {
```

Comment:
This will result in any runtime that contains a string like "version: <version>" being reported as the runc version in docker info incorrectly.

Reply:
Actually the Docker cli has the string "runc" hard-coded into the info command. I'll send a PR to address this as "runtime version" instead of "runc version".

Reply:
I think runc-commit is documented as part of the API response, see https://docs.docker.com/engine/api/v1.40/#operation/SystemInfo. I'm not sure it's as straight forward to disentangle as changing the string.